Cargo.toml

```diff
@@ -9,4 +9,4 @@ edition = "2018"
 [dependencies]
 serde_json = "1.0"
 structopt = "0.3.13"
-bewallet = { git = "https://github.com/LeoComandini/BEWallet" }
+bewallet = { git = "https://github.com/LeoComandini/BEWallet", branch = "2022-09-06-testnet" }
```

Comment:
After merge on lib remove branch and use master

Reply:
Done